libraries/helpers.rb

```diff
@@ -0,0 +1,22 @@
+module RabbitMQCookbook
+  module Helpers
+    require 'mixlib/shellout'
+
+    def remove_self_from_cluster
```

Comment:
Interesting, based on the documentation I see:

- https://www.rabbitmq.com/clustering.html
-  https://www.rabbitmq.com/man/rabbitmqctl.1.man.html
 
Based on what I read I don't see anything that suggests that it does not communicate these changes back to to cluster, is there something I am missing or do we need to enhance the documentation?